src/main/java/com/ood/myorange/controllor/StorageConfigController.java

```diff
@@ -0,0 +1,71 @@
+package com.ood.myorange.controllor;
+
+import com.ood.myorange.auth.IAuthenticationFacade;
+import com.ood.myorange.config.storage.S3Configuration;
+import com.ood.myorange.dto.StorageConfigDto;
+import com.ood.myorange.dto.response.BaseResponse;
+import com.ood.myorange.service.StorageConfigService;
+import com.ood.myorange.service.impl.StorageConfigServiceImpl;
+import com.ood.myorange.util.ModelMapperUtil;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+/**
+ * Created by Guancheng Lai on 3/19/20.
+ */
+
+@RestController
+@Slf4j
+public class StorageConfigController {
+    @Autowired
+    IAuthenticationFacade authenticationFacade; // authenticationFacade can be used to obtain the current login user
+
+    @Autowired
+    StorageConfigService storageConfigService;
+
+    @GetMapping("/view-configuration-list")
```

Comment:
view-configuration-list -> /config/storages

Reply:
Got it, accomplished